reader-sdk-react-native-quickstart/ios/RNReaderSDKSample/Info.plist

```diff
@@ -35,28 +35,14 @@
 			</dict>
 		</dict>
 	</dict>
-	<key>NSBluetoothAlwaysUsageDescription</key>
```

Comment:
Don't we need these value to be set any more?

Reply:
I think we still need them: https://developer.squareup.com/docs/reader-sdk/build-on-ios#update-your-infoplist

Reply:
Added them.